index.html

```diff
@@ -9,38 +9,223 @@
-          </ul>
-        </section>
+    <div id="background-mobile">
+      <header id="navbar-container">
+        <a class="logo" href="#">My Logo</a>
+        <nav id="navbar">
+          <div class="slice-1"></div>
+          <div class="meat"></div>
+          <div class="slice-2"></div>
+        </nav>
+      </header>
+      <section id="welcome-section">
+      
+      <div id="shapes-background">
+        <div class="headline">
+            <section id="introduction">
+              <h1 class="section-title">I'm Anderson Glad to see you!</h1>
+              <p class="headline-text">
+              I'm a software developer! I can help you build a product , feature or
+              website Look through some of my work and experience! If you like what
+              you see and have a project you need coded, don't hestiate to contact me.
+              </p>
+            </section>
+            <section id="social-media">
+              <div class="headline-supporting-text">
+                <p>LET'S  CONNECT</p>
+              </div>
+              <ul class="headline-social-media">
+                <li><a href="https://www.facebook.com/ander.cm21" target="_blank"><img src="./assets/icons/Vector-5.svg" alt="Twitter icon"></a></li>
+                <li><a href="https://www.linkedin.com/in/andersoncanales/" target="_blank"><img src="./assets/icons/Linkedin icon.svg" alt="Linkedin icon"></a></li>
+                <li><a href="https://www.facebook.com/ander.cm21" target="_blank"><img src="./assets/icons/Vector-1.svg" alt="M icon"></a></li>
+                <li><a href="https://github.com/anderCM" target="_blank"><img src="./assets/icons/Vector-4.svg" alt="Github icon"></a></li>
+                <li><a href="https://www.facebook.com/ander.cm21" target="_blank"><img src="./assets/icons/Vector.svg" alt="Fingers icon"></a></li>
+              </ul>
+            </section>
+          </div>
+        </div>
+      </div>
+    </section>
+    <section id="works-section">
+      <div class="work-container">
+        <img class="align-item" src="./assets/images/work-1.png" alt="Tonic">
+        <div class="work-info-container">
+          <h1 class="align-item">Tonic</h1>
+        </div>
+        <div class="work-description align-item">
+          <p class="company">CANOPY</p>
+          <div class="separator"></div>
+          <p class="position">Back End Dev</p>
+          <div class="separator"></div>
+          <p class="year">2015</p>
+        </div>
+        <div class="project-description align-item">
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        </div>
+        <div class="technologies align-item">
+          <div class="technologie-label">
+            <p>html</p>
+          </div>
+          <div class="technologie-label">
+            <p>css</p>
+          </div>
+          <div class="technologie-label">
+            <p>javascript</p>
+          </div>
```

Comment:
- [ ] Kindly make sure your tech stack list is grabbed inside a list of element.
`<ul>`
`<li>html</li>`
`<li>css</li>`
`</ul>`

Reply:
Thanks for feedback, Tech skills will be wrapped in a `ul` and `li` tag